src/output-json.c

```diff
@@ -438,8 +438,15 @@ void EvePacket(const Packet *p, JsonBuilder *js, unsigned long max_length)
         return;
     }
     if (!jb_set_uint(js, "linktype", p->datalink)) {
+        jb_close(js);
```

Comment:
is this a bug that also needs a backport?

Reply:
Yes, I'll create an issue for this.

Reply:
https://redmine.openinfosecfoundation.org/issues/7194